python/metric_pipeline_tasks/MatchedCatalogsAggregation.py

```diff
@@ -41,3 +41,23 @@ class MatchedCatalogsAggregationTask(CatalogsAggregationBaseTask):
 
     ConfigClass = MatchedCatalogsAggregationTaskConfig
     _DefaultName = "matchedCatalogsAggregationTask"
+
+
+class MatchedCatalogsTractAggregationTaskConnections(CatalogsAggregationBaseTaskConnections):
+    measurements = pipeBase.connectionTypes.Input(doc="{package}_{metric}.",
+                                                  dimensions=("tract", 
```

Comment:
I would like to figure out if we can just change the dimensions in a sub-class of the task or if we really need all this boilerplate, but maybe that's not for this ticket.  If you decide not to handle it here, will you file a jira issue in our epic to look into this?

Reply:
I think you're right, but I'm gonna separate it onto a new ticket: [DM-26573](https://jira.lsstcorp.org/browse/DM-26573)